pkg/testing/util.go

```diff
@@ -186,40 +187,51 @@ func prettyPrintAction(action kubetesting.Action) string {
 	//
 	// 2) You can't explicitly signal that they're the same in the switch case
 	// because Go does not allow 'fallthrough' in type switches.
-	switch verb {
+	switch action := a.(type) {
```

Comment:
Does the new client-go change such that UpdateAction/CreateAction and GetAction/DeleteAction no longer have indistinguishable interfaces? If so, let's clean up the comment above this switch case about using a string instead of type switch.